core/test/unit/services/url/Resources_spec.js

```diff
@@ -105,7 +105,6 @@ describe('Unit: services/url/Resources', function () {
                     'author',
                     'primary_author',
                     'primary_tag',
-                    'url'
```

Comment:
@kirrg001 checked UrlService/UrlGenerator/Urls and their test suites. They don't seem to use specifically resources url properties, but it's extremely hard to tell without knowing the internals 100%. Would worth if you double check as well

Reply:
ok